ansible/roles/test/tasks/bgp_gr_helper/routes_update_match.txt

```diff
@@ -0,0 +1,3 @@
+r, ".*Remove route.*"
+r, ".*Create route.*"
+r, ".*Set route.*"
```

Comment:
Can you please describe what does this rule describe?

Reply:
When graceful restart is performed on VM, the peer device (SONiC) should preserve the routes. None of the log related with removing or adding route should be observed in /var/log/syslog.

Reply:
But here (if I understand right) you expect (following logic from the previous comment) to see "Remove route", "Create route", and "Set route" messages.
Did I miss something?

Reply:
The exact logic is that: If any of the below patterns is observed in /var/log/syslog while testing graceful restart, fail the test:
* `.*Remove route.*`
* `.*Create route.*`
* `.*Set route.*`